Host a Static Website with Hugo and NGINX.md

```diff
@@ -119,7 +118,7 @@ One to the things you may want to do is add a subdomain to your domain and have
   <img src="/images/AAAA.jpg" alt="AAAA records" /> 
 </p>
 
-In the section for A/AAAA Records select the link 'Add a new A record'.  For Hostname add `http://blog.example.com`. In the IP Address field enter the IP address of your Linode server. 
+In the section for A/AAAA Records select the link "Add a new A record".  For Hostname add `http://blog.example.com`. In the IP Address field enter the IP address of your Linode server. 
```

Comment:
The name of some things can be quoted with double quotes.

Reply:
Agree.